src/App.jsx

```diff
@@ -1,3 +1,22 @@
+import { Route, Switch } from 'react-router-dom';
+import Detail from './components/Detail/Detail';
+import List from './components/List/List';
+import Home from './views/Home';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <>
+      <Switch>
+        <Route exact path="/detail/:id">
+          <Detail />
+        </Route>
+        <Route exact path="/list">
+          <List />
+        </Route>
+        <Route path="/">
+          <Home />
+        </Route>
```

Comment:
If you would like to render the list page on home, you can use /:id to render the detail page directly.